esi_leap/resource_objects/test_node.py

```diff
@@ -42,8 +42,17 @@ def get_lease_uuid(self):
     def get_lessee_project_id(self):
         return self._project_id
 
+    def get_node_power_state(self):
+        return 'Off'
+
+    def get_node_provision_state(self):
+        return 'available'
+
     def set_lease(self, lease):
         return
 
     def remove_lease(self, lease):
         return
+
+    def _get_node_attr(self, attr):
+        return None
```

Comment:
Is this still needed?

Reply:
I removed it since it's not called anywhere.